.golangci.yml

```
@@ -81,6 +81,8 @@ linters-settings:
   errcheck:
     check-type-assertions: true
     check-blank: true
+  exhaustive:
+    default-signifies-exhaustive: true
```

Can you please add a link to the docs?

---

```suggestion
    # https://golangci-lint.run/usage/linters/#exhaustive
    default-signifies-exhaustive: true
```

wasn't sure if you meant here or in the config. assumed the latter